app/jobs/webhook_deliveries/deliver_job.rb

```diff
@@ -25,7 +25,7 @@ def perform(event_type, course, resource)
 
       http = Net::HTTP.new(uri.host, uri.port)
       http.read_timeout = Rails.application.secrets.webhook_read_timeout
-      http.use_ssl = (uri.scheme == 'https' && !Rails.env.development?)
+      http.use_ssl = (uri.scheme == 'https' && Rails.env.development?)
```

Comment:
@bodhish Why was thing changed? This looks wrong.

Reply:
Fixed